### DISCUSSION TITLE
我选择了 Console 适配器和 FastAPI 驱动器，但终端并没有任何反映？

### DISCUSSION THREAD

**Discussion by Illusionna** [nonebot/discussions#434] [问答]
![1](https://github.com/nonebot/discussions/assets/109838848/c411543f-4d6f-48a0-9307-89b90ff404fe)
![2](https://github.com/nonebot/discussions/assets/109838848/ea2b5a4f-2c02-4a97-82bc-a2fdfb4afbb5)
![3](https://github.com/nonebot/discussions/assets/109838848/016f8236-4661-4156-82a8-26537d57a39c)

请问这是什么情况？

**Comment by Illusionna** [comment]
而且重新创建 robot 项目，再输入 echo 插件后还会报错.
![b (2)](https://github.com/nonebot/discussions/assets/109838848/363ebfdd-513c-4782-99cc-760626407b0c)
![b (1)](https://github.com/nonebot/discussions/assets/109838848/a3f85fcc-80be-40c1-9e5e-cccee3560716)

**Comment by yanyongyu** [comment]
你没有选择console适配器啊
